blog/index.md

```diff
@@ -5,10 +5,17 @@ nav:
   tooltip: Musings and miscellany
 ---
 
-# <i class="fas fa-feather-alt"></i>Blog
+# {% include icon.html icon="fa-solid fa-feather-pointed" %}Blog
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
Looks like there's still some placeholder text here. Screenshot from the "blog" main page:

![Screenshot from 2023-03-21 15-14-16](https://user-images.githubusercontent.com/2345877/226717106-6ee5cbd6-9673-4782-98a3-e65cb780e399.png)

Reply:
Oh, wait, I just noticed the responsive-ness worked here. Maybe you can disregard that comment, it could just be a quirk of my system/how I resized the pages.

Reply:
That is indeed intentional. It's a "big" variant of the header, which only shows up on the homepage to be "bolder", so-to-speak. All the other pages are "single row" as they are in the original site.